### DISCUSSION TITLE
検索でコメント数の指定をしたい

### DISCUSSION THREAD

**Discussion by tsubolabo** [increments/qiita-discussions#332] [要望・アイデア・不具合]
現在、ストック数（stocks:）での検索は可能ですが、いいね数やコメント数による検索はできそうにありません。いいね数については、検索後のリストをいいね数の降順に並べ替える機能があるので、これで補えます。しかし、コメント数については代替方法はなさそうです。

　コメントには賞賛コメントもあれば、指導コメントも、批判コメントもあります。賞賛や指導のコメントは、大いに役つことばかりで、検索されても心配はありません。批判コメントは、検索されることで炎上を助長する危険はありますが、自分が気づいていなかった問題に気付かせて貰えたりもするので、悪いことばかりではありません。

　ということで、コメント数による検索オプションの追加を提案します。

**Comment by tanaka-asami415** [comment]
@tsubolabo 
ありがとうございます🙏
検索関連のプロジェクトがあるので、今回のご意見も参考にさせていただきます！
